file-formats/gsmp/examples/z_aff_example_gsmp.gsmp.json

```diff
@@ -0,0 +1,20 @@
+{
+	"formatVersion": "1",
+	"header": {
+		"description": "AFF Example",
+		"originalLanguage": "en",
+		"abapLanguageVersion": "standard"
```

Comment:
I think it's important to show the language version. Only Execution Mode 'A' providers may be implemented in ABAP language version 5. The rest (Execution Mode S,I and N) need to be implemented in 0

Reply:
I think @wurkza wants to express the field `abapLanguageVersion` is serialised in the JSON as long as it has another value then `standard`. If the object has `"abapLanguageVersion": "cloudDevelopment"`, the content is serialized. 

This means, it could be distinguished whether an object has ABAP language version "standard" or "cloudDevelopment" (aka 5).